package.json

```diff
@@ -1,9 +1,10 @@
 {
   "private": true,
   "name": "karrot-frontend",
-  "productName": "Karrot",
+  "siteName": "Karrot",
   "version": "9.4.0",
-  "description": "Modern website for organization of foodsaving groups worldwide",
+  "title": "Karrot - Start a group, become a community",
```

Comment:
`title` field isn't defined as part of [`package.json` spec](https://docs.npmjs.com/cli/v7/configuring-npm/package-json), or [quasars spec](https://quasar.dev/quasar-cli/quasar-conf-js#property-build).

Seems the best way to define extra properties for using in the html template is like this:
https://quasar.dev/quasar-cli/quasar-conf-js#property-htmlvariables

Reply:
Hey @nicksellen  thanks for the review, think that i just added a new parsing to quasar instead of using its own function